src/components/Map.js

```diff
@@ -84,39 +79,8 @@ const Map = ({
         />
       </div>
       <div className="map" id="map" style={{ height: `${height}px` }}></div>
-      {legendVisible ? (
-        <div className="map-layers d-print-none">
-          <div className="close-box" onClick={toggleLegendVisibility}>
-            &minus;
-          </div>
-          <div>
-            <div
-              className="map-legend-item"
-              title="paved, separated (off the street) bikeways"
-            >
-              <div className="map-legend-icon class1"></div>
-              <label>Multi-use Path</label>
-            </div>
-            <div
-              className="map-legend-item"
-              title="dedicated on-street bikeways, marked by striping on pavement"
-            >
-              <div className="map-legend-icon class2"></div>
-              <label>Bike Lane</label>
-            </div>
-            <div
-              className="map-legend-item"
-              title="on-street routes signed for bicyclists"
-            >
-              <div className="map-legend-icon class3"></div>
-              <label>Bike Route</label>
-            </div>
-          </div>
-        </div>
-      ) : (
-        <div className="map-layers-open-box" onClick={toggleLegendVisibility}>
-          Toggle Map Legend
-        </div>
+      {isMobile !== undefined && (
```

Comment:
New conditional here to only render if `isMobile !== undefined`

Reply:
This would be better to just use a falsey check like `!isMobile`. FWIW "undefined" is never actually a state like you've compared here, though browsers handle it gracefully.